analyzers/tests/SonarAnalyzer.UnitTest/Helpers/AspNetMvcHelperTest.cs

```diff
@@ -127,5 +127,22 @@ public class Foo : Microsoft.AspNetCore.Mvc.ControllerBase
 
             AspNetMvcHelper.IsControllerMethod(publicFoo).Should().Be(false);
         }
+
+        [DataTestMethod]
```

Comment:
I think we miss a test with a methodKind != Ordinary. e.g. public constructor.

Reply:
Added!